### PR TITLE
Update all files using Mini license header to use full ASF header

### PR DESCRIPTION
**Pull request by mrutkows**

### CODE REVIEW

File: commands/action.go [hunk 1]
```diff
@@ -40,7 +40,7 @@ import (
 
 const (
 	MEMORY_LIMIT      = 256
-	TIMEOUT_LIMIT     = 60000
+	TIMEOUT_LIMIT     = 600000
```
Comment (rabbah) [inline commands/action.go]: slipped in?

File: more [hunk 2]
```diff
@@ -0,0 +1,13 @@
+./commands/property.go:		// get [1;36;40m[Kcurrent[m[K props
```
Comment (rabbah) [inline more]: delete this file?

File: commands/action.go (hunk 1 above)
Comment (mrutkows) [reply commands/action.go]: was playing with a month or so ago...  likely on my master... good catch

File: more (hunk 2 above)
Comment (mrutkows) [reply more]: all better... was seeing what would happen if client limit value exceeded server-side...

**Pull request merged**